apps/src/lib/script-editor/LevelTokenDetails.jsx

```diff
@@ -80,12 +88,21 @@ export class UnconnectedLevelTokenDetails extends Component {
     levelKeyList: PropTypes.object.isRequired,
     chooseLevel: PropTypes.func.isRequired,
     addVariant: PropTypes.func.isRequired,
+    removeVariant: PropTypes.func.isRequired,
     setActiveVariant: PropTypes.func.isRequired,
     setField: PropTypes.func.isRequired,
     level: levelShape.isRequired,
     stagePosition: PropTypes.number.isRequired
   };
 
+  constructor(props) {
```

Comment:
if you're just setting initial state here, you can remove the constructor and set state like:

```
state = {
  showBlankProgression: false
};
```